proto/proto/test_context.proto

```diff
@@ -4,5 +4,35 @@ import "google/protobuf/timestamp.proto";
 
 package test_context.test_run;
 
+enum TestCaseRunStatus {
+  TEST_CASE_RUN_STATUS_UNSPECIFIED = 0;
+  TEST_CASE_RUN_STATUS_SUCCESS = 1;
+  TEST_CASE_RUN_STATUS_FAILURE = 2;
+  TEST_CASE_RUN_STATUS_SKIPPED = 3;
+}
+
 message TestCaseRun {
+  string id = 1;
+  string name = 2;
+  string classname = 3;
+  string file = 4;
+  // need parent_id
+  string parent_name = 5;
+  int32 line = 6;
+  TestCaseRunStatus status = 7;
+  int32 attempt_number = 8;
+  google.protobuf.Timestamp started_at = 9;
+  google.protobuf.Timestamp finished_at = 10;
+  string status_output_message = 11;
+}
+
+message UploaderMetadata {
+    string version = 1;
+    string origin = 2; // RSpec, jest, etc..
```

Comment:
Should this include version info too?

Reply:
Doesn't it above?

Reply:
Nevermind, I parsed version there as a schema version, I was referring to origin version (Rspec plugin 1.0.0, etc.). This should be fine